charts/uptime-kuma/templates/statefulSet.yaml

```diff
@@ -83,11 +83,19 @@ spec:
           {{- end }}
         {{- if .Values.extraCertificates.enabled }}
         - name: cacerts
+          {{- if .Values.extraCertificates.existingConfigMap }}
+          configMap:
+            name: {{ .Values.extraCertificates.existingConfigMap }}
+            items:
+              - key: {{ .Values.extraCertificates.existingConfigMapKey | default "cacerts.pem" }}
```

Comment:
Unless I am wrong, but I think the `default` is obsolete here, as the value from `values.yaml` will be evaluated as the default, even if you try to set the value as an _empty_ value.

Reply:
Yeah, you might be right. TEchnically you could set the key to `""`. In that case the default value would be used here. But that might not be necessary, yeah.